flume-ng-channels/flume-file-channel/src/test/java/org/apache/flume/channel/file/TestFileChannelErrorMetrics.java

```diff
@@ -159,8 +161,8 @@ public void testCorruptEventTaken() throws Exception {
 
   @Test
   public void testCheckpointWriteErrorCount() throws Exception {
-    int checkpointInterval = 1000;
-    FileChannel channel = createFileChannel(Collections.singletonMap(
+    int checkpointInterval = 1500;
```

Comment:
I increased this to 1500 because 1000 made the test flaky: it might have happened that the checkpoint was created in the same second when the checkpoint writer thread updated it which caused the `lastModified` check to fail on OSes/file systems where the lastmod timestamp has second granularity.